tests/simulator/refinement/harris.py

```
@@ -0,0 +1,225 @@
+#!/usr/bin/env python3
+
+import pyphare.pharein as ph  # lgtm [py/import-and-import-from]
+from pyphare.pharein import Simulation
+from pyphare.pharein import MaxwellianFluidModel
+from pyphare.pharein import (
+    ElectromagDiagnostics,
+    FluidDiagnostics,
+    ParticleDiagnostics,
```

Ruff

`pyphare.pharein.ParticleDiagnostics` imported but unused

Remove unused import: `pyphare.pharein.ParticleDiagnostics`

(F401)